shadows/framework/src/main/java/org/robolectric/shadows/ShadowPausedAsyncTask.java

```diff
@@ -3,7 +3,7 @@
 import static org.robolectric.util.reflector.Reflector.reflector;
 
 import android.os.AsyncTask;
-import androidx.test.annotation.Beta;
+import com.google.common.annotations.Beta;
```

Comment:
I've used that one, because it's already used in the project.
But `androidx.test.annotation.Beta` suggests to use `androidx.test.annotation.ExperimentalTestApi` instead.

I can switch to that one instead (or something else), but I don't if there are some tooling/checks/... that checks for `com.google.common.annotations.Beta` at Google.